specification/ai/Azure.AI.Projects/agents/vector_stores/common/main.tsp

```diff
@@ -87,3 +87,46 @@ model VectorStoreStaticChunkingStrategyResponse
   /** The options for the static chunking strategy. */
   static: VectorStoreStaticChunkingStrategyOptions;
 }
+
+/** Type of vector storage configuration. */
+union VectorStorageDataSourceAssetType {
+  uri_asset: "uri_asset",
+  id_asset: "id_asset"
+}
+
+/**
+ * The structure, containing Azure asset ID and the asset type of the file used as a data source
+ * the enterprise file search.
+ */
+model VectorStorageDataSource {
+  /** Asset URI. */
+  @encodedName("application/json", "uri")
+  storageUri: string;
+
+  /** The asset type **/
+  @encodedName("application/json", "type")
+  assetType: VectorStorageDataSourceAssetType;
+}
+
+/**
+ * Vector storage configuration is the list of data sources, used when multiple
+ * files can be used for the enterprise file search.
+ */
+model VectorStorageConfiguration {
+  /** Data sources */
+  @encodedName("application/json", "data_sources")
+  dataSources: VectorStorageDataSource[]
+}
+
+/**
+ * The structure, containing the list of vector storage configurations i.e. the list of azure asset IDs.
+ */
+model VectorStoreAzureConfigurations {
```

Comment:
Please rename to VectorStoreConfigurations

Reply:
Fixed.